### DISCUSSION TITLE
年紀越大膽子越小

### DISCUSSION THREAD

**Discussion by WeiYen1103** [1101-GS4719-C/Python-Community-GS4719-C#1759] [General]
最近跟朋友聊天突然聊到，
ㄟ你有坐過六福村的笑傲飛鷹嗎，我回他: 沒有ㄟ，
但現在叫我去坐我也不敢去了，回想起小學時的我，根本天不怕地不怕，去樂園都挑最恐怖的坐，家人都覺得我超級勇敢；但到了現在，也不知道是不是因為看了許多的新聞，知道了有些遊樂設施的可怕之處，開始體悟到生命的脆弱，漸漸地害怕一些以前敢做或是未做的事情，進而變得惜命變得膽小，不敢踏出自己畫的這個舒適圈，不敢嘗試好多新的事物。

**Comment by Patsy17** [comment]
真的！以前很多敢做的事情，隨著年紀越來越大後，越來越不喜歡做些刺激的事，
感覺是因為我會想更多可能的後果，然後就會害怕。

**Comment by mrweixd** [comment]
小的時候真的不會想太多。我還記得以前敢瘋狂去徒手抓螞蟻，現在連碰都不想碰。

**Comment by davidteng00000** [comment]
人隨年紀增長，對風險的意識可能增強。從過去勇敢的行為到現在謹慎，這種轉變很自然。認識到生命脆弱，擔心風險是普遍現象。嘗試新事物需要勇氣，也是成長的一部分。

**Comment by 1chooo** [comment]
真的！不過有時候長大了還可以做小時候會做的事，反而讓人特別滿足！？（算是知足的一種展現嗎⋯⋯）

**Comment by 111426027** [comment]
應該是年紀增長後，顧慮的事情會比較多，因此沒有辦法像年輕的時候放手一搏的去做一件事情吧。

**Comment by yuchien-lin** [comment]
我發現我也是，小時候非常的大膽，想做什麼馬上就去做，現在做事都要思考一下，能不能做對自己有沒有害才去執行，不過這也反映了我們大腦思考的能力越來越好，在以前我們沒有看過這麼多的遊樂設施的問題，長大後懂得思考一下這會不會有危險，這應該不算壞事啦。

**Comment by B1nn9** [comment]
對遊樂設施的部分我反而相反，以前小時候都會腦補從遊樂設施上掉下來會不會很痛死掉之類的然後就不敢玩，但長大之後就開始意識到其實沒有那麼誇張，況且定期都有人在檢修，每天也都不少人在玩，要那麼衰應該也不會輪到我。所以後來在遊樂園就越來越敢玩了，以前覺得恐怖的設施都會一個個解鎖😆

**Comment by William08290829** [comment]
對於遊樂設施我也覺得我是相反，反而長大會更敢玩以前小時候不敢玩的360那種之類的，就意識到玩了也不會死所以就感覺沒這麼可怕了。我覺得可能真的要等到中年程度才會覺得年紀越大膽子越小的問題ww

**Comment by jeff110303549** [comment]
這種變化是很自然的，隨著年齡的增長和生活經驗的累積，人們對於風險和危險的認知也會有所改變。年少時的無畏和勇氣可能會隨著成長而漸漸被珍惜生命和安全的意識所取代。這並不是膽小，而是對於生命更加珍惜和負責任的表現。
每個人的成長過程都會有不同的轉變和體悟，重要的是能夠接受自己的改變，並且在安全的範圍內探索和嘗試新的事物。

**Comment by EPHMR0524** [comment]
天啊這根本就是我，小的時候去遊樂園玩總是跟父母說要去玩那些最恐怖的遊樂設施，但因為身高不夠總是被拒於門外但是現在身高到了也是不敢去玩，可能是老了吧。

**Comment by 110101501** [comment]
小時候不管做什麼是都可以不顧一切地做。
長大後在做每件事之前往往都要想一下後果、會有怎麼風險、社會會怎樣看待我。
種種面向，導致在下定決定做之前，會有很多考量，最後可能就不願意、不敢去做了。
